pkg/operator/apiservicecontroller/apiservice_controller.go

```diff
@@ -32,6 +40,7 @@ const (
 type APIServiceController struct {
 	name        string
 	apiServices []*apiregistrationv1.APIService
+	apiGroupsManagedByOAuthServer []schema.GroupVersion
```

Comment:
would use more generic names. We will have another split eventually.

Reply:
like `apiGroupsManagedByExternalServer` ?